features/step_definitions/backend_api_steps.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+Given /^a backend api "([^"]*)"( that is unnaccessible)?$/ do |name, unnaccessible|
```

Comment:
(Apart from the typo) "accessible"/"inaccessible" was a very unfortunate choice or ours in the code to distinguish if a record is active or been deleted. I wouldn't expose this mistake in the cucumber step.

```suggestion
Given /^a backend api "([^"]*)"( that is deleted)?$/ do |name, deleted|
```